LaunchShowcase.Shared/LaunchShowcase.Shared.projitems

```diff
@@ -28,8 +28,17 @@
       <DependentUpon>MainPage.xaml</DependentUpon>
     </Compile>
     <Compile Include="$(MSBuildThisFileDirectory)TemplateSelectors\ShowcasePageTemplateSelector.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Themes\ShowcaseTemplates\FluentStoreControl.xaml.cs">
```

Comment:
Looks like you accidentally overwrote some code needed for another template page, causing builds on this branch to fail. Could you please fix this?